plugin/storage/es/mappings/fixtures/jaeger-service-with-ilm-7.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
how different is it from `-7`? I would rather use some template / substitution, not duplicate a pretty large file